src/evo/deterministicmns.h

```diff
@@ -40,18 +40,24 @@ class CDeterministicMN
     uint64_t internalId{std::numeric_limits<uint64_t>::max()};
 
 public:
+    static constexpr uint16_t TYPE_REGULAR_MASTERNODE = 0;
+    static constexpr uint16_t TYPE_HIGH_PERFORMANCE_MASTERNODE = 1;
```

Comment:
make this an enum (I think not enum class here, as enum class cannot easily be converted to underlying type